docs: mention running the CLI after installation (#291)
apify/crawlee-python: docs/introduction/01-setting-up.mdx

pipx run crawlee create my-crawler
```

> If you already have crawlee installed . Then you can spin up crawlee by 

Reviewer: ```suggestion
If you already have `crawlee` installed, you can spin it up by running:

```

```sh

> crawlee create mycrawler

Reviewer: ```suggestion
crawlee create my-crawler
```
